ui/src/literature/utils.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/prefer-default-export
+export function getPageDisplay(pagesInfo) {
```

Comment:
I would create separate tests for this, since it's really easy to tests, although I get that it's tested through other ones.